### PR TITLE
Config file to deploy OCP 4.4 GAed version

### PR DESCRIPTION
**Pull request by Shrivaibavi**
Signed-off-by: Shrivaibavi Raghaventhiran <sraghave@redhat.com>

### CODE REVIEW

File: conf/ocp_version/ocp-4.4-ga-config.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+RUN:
+  client_version: '4.4-ga'
+
+DEPLOYMENT:
+  ocp_url_template: "https://mirror.openshift.com/pub/openshift-v4/clients/ocp/{version}/{file_name}-{os_type}-{version}.tar.gz"
+  installer_version: "4.4-ga"
+  ocp_channel: "stable"
+ENV_DATA:
+  vm_template: 'rhcos-4.2.0-x86_64-vmware'
```
Comment (prsurve) [inline conf/ocp_version/ocp-4.4-ga-config.yaml:12]: it should be rhcos-4.4.0-x86_64-vmware

Comment (Shrivaibavi) [reply conf/ocp_version/ocp-4.4-ga-config.yaml]: Done

File: conf/ocp_version/ocp-4.4-ga-config.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+RUN:
+  client_version: '4.4-ga'
+
+DEPLOYMENT:
+  ocp_url_template: "https://mirror.openshift.com/pub/openshift-v4/clients/ocp/{version}/{file_name}-{os_type}-{version}.tar.gz"
+  installer_version: "4.4-ga"
+  ocp_channel: "stable"
+ENV_DATA:
+  vm_template: 'rhcos-4.4.0-x86_64-vmware'
```
Comment (vavuthu) [inline conf/ocp_version/ocp-4.4-ga-config.yaml:12]: Where do u get above template ?

Comment (Shrivaibavi) [reply conf/ocp_version/ocp-4.4-ga-config.yaml:12]: I used the template from ocp-4.3-ga-config.yaml and ocp-4.2-ga-config.yaml

Comment (petr-balogh) [reply conf/ocp_version/ocp-4.4-ga-config.yaml:12]: https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.4/latest/ there is no such image yet so go with what we have in non GA and make a comment/note about once there will be RHCOS gaed image we should change here and open the issue to ecosystem team please about this which you can mention in the comment.

Thanks

Comment (Shrivaibavi) [reply conf/ocp_version/ocp-4.4-ga-config.yaml]: Thanks @petr-balogh I changed the version and also added comment once GAed we can change that.

File: conf/ocp_version/ocp-4.4-ga-config.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+RUN:
+  client_version: '4.4-ga'
+
+DEPLOYMENT:
+  ocp_url_template: "https://mirror.openshift.com/pub/openshift-v4/clients/ocp/{version}/{file_name}-{os_type}-{version}.tar.gz"
+  installer_version: "4.4-ga"
+  ocp_channel: "stable"
+ENV_DATA:
+  vm_template: 'rhcos-4.4.0-rc.1-x86_64-vmware.x86_64'
```
Comment (clacroix12) [inline conf/ocp_version/ocp-4.4-ga-config.yaml:12]: Should this use the template without `rc` in the name as described below?

Comment (clacroix12) [reply conf/ocp_version/ocp-4.4-ga-config.yaml:12]: Nevermind I missed the comment from Petr, looks like this isn't available yet.

**Comment by Shrivaibavi** [comment]
Just FYI, The deployment job passed with ocp 4.4-ga parameter
https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/7260/

**Pull request merged**